docs/OrganizingCommittee/Committee-formation.md

```diff
@@ -0,0 +1,84 @@
+# Committee Formation
+
+Usually a hack week is spearheaded by one or several individual who have the motivation to host a hack week for a specific community. Once topic of a hackweek is formulated, the process of gathering together other committee memebers should begin, ideally 8 - 12 months prior to the event. 
+
+Below we list the typical roles across an organizing committee. We note that for some events, certain roles might not be considered an integral part of the formal organizing committee, for example the tutorial leads. However, we believe there are several advantages to including these people at the decision making level of the overall event. These include an increased sense of ownership that results in organizers putting forth their best effort in designing and delivering content, and a broadening of the collective wisdom of the organizing committee around important decisions due to the presence of a diversity of perspectives. Early career faculty and researchers also often report that being recognized at the organizational level helps their resumes when it comes to applying for jobs. 
+
+## Tutorial Leads
+
+Generally the largest group of organizers are focused around development and delivery of data science tutorials. Responsibilities include:
+
+* polling their respective communities to determine what are the community needs around data science education
+* talking to open source software developers to connect with the most up-to-date tools and libraries available
+* building a set of learning objectives, designing and creating a tutorial
+* review the content of a tutorial and integrate it with themes of other tutorials offered in the event
+* work with cloud computing organizers to ensure all the tools are in place for participants to run interactive tutorials
+* practice the delivery of tutorials in advance of the event
+
+Additional guidelines on the roles and responsibilities for tutorial leads can be found [here](https://geohackweek.github.io/wiki/tutorial_leads.html).

```

Comment:
I have been told at one point that one shouldn't put links as "here", but rather a more descriptive text (maybe "On the GeoHackWeek wiki page for tutorial leads"?). This is related to how screen readers access web pages and read it to vision-impaired users: https://accessibility.oregonstate.edu/descriptivelinks